webapp/src/actions.js

```diff
@@ -33,7 +33,7 @@ export function setShowRHSAction(showRHSPluginAction) {
 export const getPluginServerRoute = (state) => {
     const config = getConfig(state);
 
-    let basePath = '/';
+    let basePath = '';
```

Comment:
Weird change, but I see that the `/` served no purpose here, and is indeed the bug.